### ISSUE TITLE
Pretty printing and charts for (end) results

### ISSUE THREAD

**Issue by noracato** [quintel/meurit-py#4]
Charts and nicely printed data will give users more insight in the end results. We should add a module that can show both a `Country`'s, a group of countries, and the overall data in an understandable format.

As @ChaelKruip is very good with `matplotlib`, maybe you can help a bit with the design here 🙂

**Comment by ChaelKruip** [comment]
> As @ChaelKruip is very good with matplotlib, maybe you can help a bit with the design here 🙂

Yes! I can also make some plot routines if you give me the data-formats (or, even better, some dummy data).
